packages/react-ui-ag/src/Gmap/Markers.js

```diff
@@ -2,29 +2,25 @@ import React, { PureComponent } from 'react'
 import PropTypes from 'prop-types'
 import get from 'lodash/get'
 import { Markers as Gmarkers } from '@rentpath/react-ui-core'
-import {
-  redDotIcon,
-  greyDotIcon,
-  blackDotIcon,
-} from './markerIcons'
+import MarkerIcons from './markerIcons'
 
 const NOOP = () => ({})
 
 export default class Markers extends PureComponent {
   static propTypes = {
     marker: PropTypes.func,
-    markerIcon: PropTypes.func,
-    markerInactiveIcon: PropTypes.func,
-    markerIconHover: PropTypes.func,
+    markerIcon: PropTypes.string,
```

Comment:
Since these are now limited to the icons we have defined, we may want this to be a `oneOf[...]` instead of an open string.

Reply:
Good point - I will make that change